quickstart/galoy/docker-compose.yml

```diff
@@ -102,7 +102,7 @@ services:
     environment:
       - MONGODB_ADDRESS=mongodb
   mongodb:
-    image: mongo:${MONGODB_VERSION:-6.0.5}
+    image: mongo:${MONGODB_VERSION:-7.0.2}
```

Comment:
this should not be updated manually. It happens in CI.

Reply:
don't think there is any harm is updating them manually?

but it's another argument to not have the computed quickstart in the same branch but in another branch. it make search/replace harder in practive if we want to enforce the rule of "this should not be updated manually"

Reply:
I remove the update.. just in case https://github.com/GaloyMoney/galoy/pull/3301/commits/ca426c4de10bd79733d73deb51d87b5bd541469f